stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -441,6 +443,48 @@ extension ExecutorJob {
   }
 }
 
+// Helper to create a trampoline job to execute a job on a specified
+// executor.
+extension ExecutorJob {
+
+  /// Create a trampoline to enqueue the specified job on the specified
+  /// executor.
+  ///
+  /// This is useful in conjunction with the `Clock.run()` API, which
+  /// runs a job on an unspecified executor.
+  ///
+  /// Parameters:
+  ///
+  /// - to executor:   The `Executor` on which it should be enqueued.
+  ///
+  /// Returns:
+  ///
+  /// A new ExecutorJob that will enqueue the specified job on the specified
```

Comment:
```suggestion
  /// - A new ExecutorJob that will enqueue the specified job on the specified
```

needs the `-` I think?